tests/subsys/canbus/isotp/conformance/testcase.yaml

```diff
@@ -5,3 +5,36 @@ tests:
       - isotp
     depends_on: can
     filter: dt_chosen_enabled("zephyr,canbus") and not dt_compat_enabled("kvaser,pcican")
+  canbus.isotp.conformance.fd.txdl_8:
+    tags:
+      - can
+      - isotp
+    extra_configs:
+      - CONFIG_CAN_FD_MODE=y
+      - CONFIG_ISOTP_USE_CAN_FD=y
+      - CONFIG_ISOTP_TX_DL_MAX=8
+    platform_allow:
+      - native_posix
+      - native_posix_64
```

Comment:
Is there a twister filter like `dt_chosen_enabled("zephyr,canbus")` for CAN-FD?
For now, I only enabled ISO-TP CAN-FD testing on native posix

Reply:
No, you cannot filter tests based on CAN-FD support. You'll need to skip tests if runtime CAN-FD support is not available.

Reply:
Good idea, I have made that change